addons/compat_spe/compat_spe_csw/CfgWeapons.hpp

```diff
@@ -1,26 +1,145 @@
 class CfgWeapons {
-    class Launcher;
-    class Launcher_Base_F: Launcher {
-        class WeaponSlotsInfo;
+    class Launcher_Base_F;
+    class SPE_Slung_Static_Weapon_Base: Launcher_Base_F {};
+
+    // --- Vanilla Compat -------------------------------------------------------------
+    class EGVAR(csw,staticHMGCarry): Launcher_Base_F {
+        class ace_csw {
+            class assembleTo {
+                GVAR(M3_Tripod) = "B_HMG_01_F";
```

Comment:
it's a little strange that this is a one way conversion because B_HMG_01_F won't disassemble back to this tripod
but I think maximizing compatibility is more important

Reply:
Yea, I'm not a fan of the current solution, but it's better than nothing.

I was trying to think of a better solution to this problem and have come up with two:
1) Either set a variable on the static that specifies the type of bipod and weapon, which would override the config stuff.
2) Or make a static class per combination. That increases the amount of classes by n^2, so imo the easier solution is 1)

Reply:
1) was done in #10300